1.2/sd-client/pom.xml

```diff
@@ -0,0 +1,145 @@
+    <artifactId>sd-common-pom</artifactId>
+    <version>1.0.5</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>sd-client</artifactId>
+  <name>Service Directory Client Application</name>
+  <version>1.2.1-0-SNAPSHOT</version>
+  <description>Project for Cisco OSS service directory Client Application.</description>
+  <url>https://github.com/foundation-runtime/service-directory/sd-client</url>
+  <packaging>jar</packaging>
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+  <issueManagement>
+    <system>Github
+        </system>
+    <url>{URL in JIRA or Product name in CQ}</url>
+  </issueManagement>
+  <mailingLists>
+    <mailingList />
+  </mailingLists>
+  <developers>
+    <developer>
+      <id>Alex Wu</id>
+      <name>Yiding Wu</name>
+      <email>yidwu@cisco.com</email>
+      <organization>Cisco</organization>
+      <organizationUrl>http://www.cisco.com</organizationUrl>
+      <roles>
+        <role>Developer</role>
+      </roles>
+    </developer>
+    <developer>
+      <id>Xuan Zhong</id>
+      <name>Xuan Zhong</name>
+      <email>xuzhong@cisco.com</email>
+      <organization>Cisco</organization>
+      <organizationUrl>http://www.cisco.com</organizationUrl>
+      <roles>
+        <role>Developer</role>
+      </roles>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:git@github.com:foundation-runtime/service-directory.git</connection>
+    <developerConnection>scm:git:git@github.com:foundation-runtime/service-directory.git</developerConnection>
+    <url>https://github.com/foundation-runtime/service-directory.git</url>
+    <tag>HEAD</tag>
+  </scm>
+  <organization>
+    <name>Cisco</name>
+    <url>http://www.cisco.com</url>
+  </organization>
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-javadoc-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>attach-doc</id>
+            <goals>
+              <goal>jar</goal>
+            </goals>
+            <phase>package</phase>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-site-plugin</artifactId>
+      </plugin>
+    </plugins>
+  </build>
+
+  <reporting>
+    <plugins>
+      <plugin>
+        <artifactId>maven-project-info-reports-plugin</artifactId>
+        <version>2.8</version>
+     </plugin>
+      <plugin>
+        <artifactId>maven-javadoc-plugin</artifactId>
+        <version>2.10.3</version>
+      </plugin>
+      <plugin>
+        <artifactId>maven-surefire-report-plugin</artifactId>
+        <version>2.18.1</version>
+      </plugin>
+    </plugins>
+  </reporting>
+
+  <properties>
+    <sd.core.version>1.2.1-1</sd.core.version>
+    <sd.api.version>1.2.1-5-SNAPSHOT</sd.api.version>
```

Comment:
It is better not to use a snapshot version. We have some issues with the OSS snapshot repo. It would fail the build.